Sources/Configurations/AlphabeticConfiguration.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  AlphabeticConfiguration.swift
+//  FormValidatorSwift
+//
+//  Created by Aaron McTavish on 03/01/2017.
+//  Copyright © 2017 ustwo. All rights reserved.
+//
+
+import Foundation
+
+
+/// Stores configuration for `AlphabeticCondition`.
+public struct AlphabeticConfiguration: Configuration {
+    
+    
+    // MARK: - Properties
+    
+    /// Whether or not to allow Unicode letters. If `false` then only ASCII letters (A-Z, a-z) are allowed.
+    public let allowsUnicode: Bool
+    /// Whether or not to allow whitespace.
+    public let allowsWhitespace: Bool
+    
+    
+    // MARK: - Initializers
+    
+    public init() {
+        self.init(allowsUnicode: false, allowsWhitespace: false)
+    }
+    
+    public init(allowsUnicode: Bool = false, allowsWhitespace: Bool = false) {
```

Comment:
What did you think about using the struct's default parameter values before deciding on an initializer based approach? Is it about helping enforce an initialization for all configurations?